src/tests/functional/plugin/gpu/shared_tests_instances/low_precision_transformations/recurrent_cell_transformation.cpp

```diff
@@ -0,0 +1,177 @@
+        ngraph::builder::subgraph::RecurrentCellFunction::RNNType::LSTMSequence,
+        "RNNCell",
+        "FP32"
+    }
+};
+
+const std::vector<std::vector<ngraph::PartialShape>> activations_shapes = {{{1, 1, 16}, {1, 1, 128}, {1, 1, 128}}};
+const std::vector<std::vector<ngraph::Shape>> weights_shapes = {{{1, 512, 16}, {1, 512, 128}, {1, 512}}};
+
+INSTANTIATE_TEST_SUITE_P(smoke_LPT, RecurrentCellTransformation,
+    ::testing::Combine(
+        ::testing::ValuesIn(netPrecisions),
+        ::testing::ValuesIn(activations_shapes),
+        ::testing::ValuesIn(weights_shapes),
+        ::testing::Values(CommonTestUtils::DEVICE_GPU),
+        ::testing::ValuesIn(trasformationParamValues),
+        ::testing::ValuesIn(params)),
+    RecurrentCellTransformation::getTestCaseName);
+} // namespace testValues1
+
+namespace testValues2 {
+
+const std::vector<LayerTestsDefinitions::RecurrentCellTransformationParam> params = {
+    // GRUSequence
+    {
+        // X
+        {256ul, {}, {0.f}, {2.55f}, {0.f}, {255.f}},
+        {ngraph::element::u8},
+        {
+             {ngraph::element::f32},
+             {},
+             {0.01f},
+        },
+        // H
+        {256ul, {}, {0.f}, {2.55f}, {0.f}, {255.f}},
+        {ngraph::element::u8},
+        {
+             {ngraph::element::f32},
+             {},
+             {0.01f},
+        },
+        // W
+        {255ul, {}, {-1.27f}, {1.27f}, {-1.27f}, {1.27f}},
+        {},
+        {{}, {}, {}},
+        // R
+        {255ul, {}, {-1.27f}, {1.27f}, {-1.27f}, {1.27f}},
+        {},
+        {{}, {}, {}},
+        ngraph::builder::subgraph::RecurrentCellFunction::RNNType::GRUSequence,
+        "RNNCell",
+        "U8"
+    },
+    // asymmetrical FQ on weights
+    {
+        // X
+        {256ul, {}, {0.f}, {2.55f}, {0.f}, {255.f}},
+        {ngraph::element::u8},
+        {
+             {ngraph::element::f32},
+             {},
+             {0.01f},
+        },
+        // H
+        {256ul, {}, {0.f}, {2.55f}, {0.f}, {255.f}},
+        {ngraph::element::u8},
+        {
+             {ngraph::element::f32},
+             {},
+             {0.01f},
+        },
+        // W
+        {256ul, {}, {-1.27f}, {1.27f}, {-1.27f}, {1.27f}},
+        {},
+        {{}, {}, {}},
+        // R
+        {256ul, {}, {-1.27f}, {1.27f}, {-1.27f}, {1.27f}},
+        {},
+        {{}, {}, {}},
+        ngraph::builder::subgraph::RecurrentCellFunction::RNNType::GRUSequence,
+        "RNNCell",
+        "FP32"
+    }
+};
+
+const std::vector<std::vector<ngraph::PartialShape>> activations_shapes = {{{1, 2, 3}, {1, 2, 3}, {}}};
+const std::vector<std::vector<ngraph::Shape>> weights_shapes = {{{1, 9, 3}, {1, 9, 3}, {1, 9}}};
+
+INSTANTIATE_TEST_SUITE_P(smoke_LPT, RecurrentCellTransformation,
+    ::testing::Combine(
+        ::testing::ValuesIn(netPrecisions),
+        ::testing::ValuesIn(activations_shapes),
+        ::testing::ValuesIn(weights_shapes),
+        ::testing::Values(CommonTestUtils::DEVICE_GPU),
```

Comment:
Should the RecurrentCellTransformation be enabled for GPU? If no, I suggest to disable this transformation in gpu plugin and disable the corresponding tests

Reply:
I have no expertise to make such kind of decisions.
In general, does LPT try to avoid an execution of device specific transformations?
Do we have other examples?
This part was already reviewed and approved before btw.

Reply:
RecurrentCellTransformation (like any other) only makes sense if we have low precision support for the corresponding layer on a device. Otherwise, we would negatively affect loadTime without any positive performance impacts (Moreover, the performance influence might be even negative).
We have already had such situations. So I would recommend to disable this transformation for GPU. To do that we need to:
- set empty restriction for the operation;
- disable the transformation.
I found the example of the specific LPT disabling in commit history: [StridedSlice example](https://github.com/openvinotoolkit/openvino/blob/60fb05bb6a58b9de708124004e8906c34fc7b249/inference-engine/src/cldnn_engine/cldnn_transformations_pipeline.cpp#L358-L369)
Also, please add the tests to [skip_tests_config](https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/gpu/shared_tests_instances/skip_tests_config.cpp)

Reply:
Done. Please double check